conf/main/logback.xml

```diff
@@ -62,11 +62,11 @@
         <appender-ref ref="${jetty.log.appender:-MAIN}"/>
     </logger>
 
-    <logger name="ai.grakn" level="${grakn.log.level}">
+    <logger name="ai.grakn" level="${grakn.log.level:-INFO}">
```

Comment:
@fppt not sure I understand. It is just setting a default. If grakn.log.level is not set in the config then it defaults to grakn.log.level.INFO if you want this can be reverted and I think I can explicitly pass it to the migrator script